client/src/ts/trainees/InternalProjectDetail.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Typography } from "@material-ui/core";
+
+interface DetailProps {
+  name: string;
+  value: string;
+}
+
+export const InternalProjectDetail: React.FC<DetailProps> = ({ name, value }) => {
```

Comment:
This component is missing its comment.
```suggestion
/**
 * Renders a value with its corresponding name
 */
export const InternalProjectDetail: React.FC<DetailProps> = ({ name, value }) => {
```

Reply:
This component is a general component which can be used outside of the internal project context.
Maybe rename it, e.g. to Detail.